apps/src/ailab/Ailab.js

```diff
@@ -18,6 +23,21 @@ import $ from 'jquery';
  */
 const MOBILE_PORTRAIT_WIDTH = 600;
 
+function getInstructionsDefaults() {
+  var instructions = {
+    selectDataset: 'Select the data set you would like to use.',
+    dataDisplayLabel: 'Choose one column to predict.',
+    dataDisplayFeatures:
+      'Choose one or more columns as inputs to help make the prediction.',
+    selectTrainer: 'Set up the training.',
+    trainModel: 'Your model is being trained.',
+    results: 'Review the results.',
+    saveModel: 'Save the trained model for use in App Lab.'
+  };
```

Comment:
i18n-ify these?

Reply:
I was going to hold off for now while we iterate quickly on this content, especially since the rest of the content is en-only...